src/StoreApi/Routes/Checkout.php

```diff
@@ -177,6 +177,10 @@ protected function get_route_post_response( \WP_REST_Request $request ) {
 		// Persist customer address data to account.
 		$order_controller->sync_customer_data_with_order( $order_object );
 
+		// Fire woocommerce_blocks_checkout_order_processed, should work the same way as woocommerce_checkout_order_processed
+		// But we're opting for a new action because the original ones attaches POST data.
+		do_action( 'woocommerce_blocks_checkout_order_processed', $order_object->get_id(), $order_object );
```

Comment:
Let's add an `__experimental` prefix on these new actions/filters for now as a signal that it _could_ change. I'm wary about merging in new actions and filters as public apis while we're still building extensions.

Reply:
Good idea. We can perhaps review `__experimental` hooks (etc) each release/cycle (I think you suggested the idea the other day @nerrad 😄).

Alongside marking them experimental, I think it's a good idea to document them upfront (i.e. don't wait til they are stable) - we could use PHPdoc or add a markdown doc.

Reply:
PHP doc preferably for now (while also indicating it's experimental in the doc too), and then bring out to our `/docs/extensibility` folder once it moves from being experimental.